recipes/qscintilla2/meta.yaml

```diff
@@ -0,0 +1,57 @@
+
+{% set name = "qscintilla2" %}
+{% set version = "2.9.3" %}
+{% set md5 = "3b1cdbce9ef6c28cd868bcc58219f96e" %}
+{% set md5_zip = "2b0da639a8ec3d4a37c69a5539d22569" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:   QScintilla-gpl-{{ version }}.tar.gz   [unix]
+  url:  http://downloads.sourceforge.net/project/pyqt/QScintilla2/QScintilla-{{ version }}/QScintilla_gpl-{{ version }}.tar.gz  [unix]
+  md5:  {{ md5 }}   [unix]
+  fn:   QScintilla-gpl-{{ version }}.zip      [win]
+  url:  http://downloads.sourceforge.net/project/pyqt/QScintilla2/QScintilla-{{ version }}/QScintilla_gpl-{{ version }}.zip     [win]
+  md5:  {{ md5_zip }}   [win]
+
+build:
+  number: 0
+  features:
+    - vc9           [win and py27]
+    - vc10          [win and py34]
+    - vc14          [win and py35]
+  script_env:
+    - LOCALAPPDATA  [win and py27]
+
+requirements:
+  build:
+    - python     
+    - sip
+    - qt
+    - pyqt
+
+  run:
+    - python
+    - sip
+    - qt
+    - pyqt
+
+test:
+    run_test.sh   [unix]
+    run_test.bat  [win]
```

Comment:
The error you're seeing is because of this. Please remove this section, `conda-build` automatically runs these test files if it finds them in the recipe directory.